tls/extensions/s2n_server_certificate_status.c

```diff
@@ -84,7 +83,6 @@ int s2n_server_certificate_status_recv(struct s2n_connection *conn, struct s2n_s
 
 static int s2n_server_certificate_status_send_size(struct s2n_connection *conn)
 {
-    notnull_check(conn);
     if (s2n_server_can_send_ocsp(conn)) {
         return sizeof(uint8_t) + U24_SIZE + conn->handshake_params.our_chain_and_key->ocsp_status.size;
```

Comment:
I know this is existing code, but could SIZEOF_UINT24 just use U24_SIZE instead?

Reply:
I don't see a SIZEOF_UINT24 in this file. Can you clarify?

But either way I'd rather not clean up that function. I hope to delete it very soon :)

Reply:
Defined in `s2n_stuffer_network_order_test.c`. If this is going away then super.